app/assets/javascripts/discourse/tests/acceptance/sidebar-categories-section-test.js

```diff
@@ -17,6 +18,50 @@ import discoveryFixture from "discourse/tests/fixtures/discovery-fixtures";
 import categoryFixture from "discourse/tests/fixtures/category-fixtures";
 import { cloneJSON } from "discourse-common/lib/object";
 
+acceptance(
+  "Sidebar - Categories Section - suppress_uncategorized_badge enabled",
```

Comment:
Should we have another test for when the `suppress_uncategorized_badge` setting is disabled?

Reply:
It is disabled by default but let me make it more explicit in the other tests.